src/RiakClient/Models/MapReduce/KeyFilters/Not.cs

```diff
@@ -54,22 +55,21 @@ public override string ToString()
 
         public string ToJsonString()
         {
+            /*
+             * NB: JsonTextWriter is guaranteed to close the StringWriter
+             * https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/JsonTextWriter.cs#L150-L160
+             */
             var sb = new StringBuilder();
-
-            using (var sw = new StringWriter(sb))
```

Comment:
This is really weird, I know I had removed this line in the past. I wonder how it reappeared.... either that, or it never triggered the warning until today.